iaas-support/alicloud/download-releases.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
why do we need these two scripts?

Reply:
These two scripts aim to resolve downloading cf releases failed in China mainland results from bad network or other uncertain reasons.